src/core/utils.js

```diff
@@ -102,3 +106,10 @@ export function loadAddonIfNeeded(
   return fetchAddon({ slug, api: state.api })
     .then(({ entities }) => dispatch(loadEntities(entities)));
 }
+
+export function getAction(type) {
```

Comment:
Actually, if this were in `core/tracking` it could probably be part of `tracking.sendEvent`:

```js
export function sendEvent({ category, action, label, type, value } = {}) {
  if (type !== undefined) {
    action = getAction(type);  // This won't pass eslint but you know.
  }
  // ...
}